bindings/java/gradlew.bat

```diff
@@ -65,7 +65,7 @@ echo location of your Java installation.
 goto fail
 
 :execute
-@rem Setup the command line
+@rem SetupWallet the command line
```

Comment:
i think this is a mistake from replaceall? :)

Reply:
good eyes 👍